djedi/templatetags/future.py

```diff
@@ -0,0 +1,8 @@
+from django.template import Library, defaulttags
```

Comment:
This may not be needed. It *may* also break other django applications depending on the `future` library, as I'm not sure which library has the highest priority if there are more than one of them. Removing this, however,  means we'll not support Django < 1.5 anymore. But that may be OK.